mosdef_cassandra/examples/gcmc.py

```diff
@@ -3,7 +3,7 @@
 import mosdef_cassandra as mc
 
 
-def run_gcmc():
+def run_gcmc(kwargs={}):
```

Comment:
I like the addition of `kwargs` into the example functions and I agree it will make testing easier in the future.  I'm wondering how you feel about having `kwargs` and `custom_args` be a single dictionary like this:

```
def run_gcmc(kwargs={}):
    ...
    # Check if chemical potential and prop_freq are in kwargs
    # Set default values if not
    if "chemical_potentials" not in kwargs:
        kwargs["chemical_potentials"] = -35.0
    elif "prop_freq" not in kwargs:
        kwargs["prop_freq"] = 100

    ...
    mc.run(
    **kwargs
    )

"""

This doesn't make a difference in the system, but is personally a little easier for me to read.

Reply:
I did something in between. Let me know what you think. I wanted to avoid all the `if/else` steps, especially for examples with more `custom_args`, like `gemc.py`, but I agree with the critique.